index.js

```diff
@@ -73,7 +73,10 @@ var addToMailchimp = function addToMailchimp(email, fields) {
   var isEmailValid = (0, _emailValidator.validate)(email);
   var emailEncoded = encodeURIComponent(email);
   if (!isEmailValid) {
-    throw 'gatsy-plugin-mailchimp: email must be of type string and a valid email address. See README for more information.';
+    return Promise.resolve({
```

Comment:
@steebchen this change isn't actually necessary bc babel converts and compiles `src/index.js` into this file.  nbd either way bc i'll end up [rebuilding](https://github.com/benjaminhoffman/gatsby-plugin-mailchimp/blob/master/package.json#L8), which will overwrite this change anyway

Reply:
I just compiled it myself and pushed it in the same commit. Is that not what you want?